### ISSUE TITLE
Update to latest version of `Cake.Core` and `Cake.Powershell`

### ISSUE THREAD

**Issue by ssrmm** [SharpeRAD/Cake.Services#26]
We recently updated to the latest version of Cake and now see some warnings produced by the script because the versions referenced in `Cake.Services` are outdated. Could you updated the project to reference the latest versions?

```
The assembly 'Cake.Powershell, Version=0.4.6.0, Culture=neutral, PublicKeyToken=null'
is referencing an older version of Cake.Core (0.29.0).
For best compatibility it should target Cake.Core version 0.33.0.
The assembly 'Cake.Services, Version=0.3.5.0, Culture=neutral, PublicKeyToken=null'
is referencing an older version of Cake.Core (0.29.0).
For best compatibility it should target Cake.Core version 0.33.0.
```

**Comment by SharpeRAD** [comment]
Resolved in **v0.3.6** 👍

**Comment by ssrmm** [comment]
Thanks 👍
